server.go

```diff
@@ -174,6 +195,10 @@ var startupHooks []func()
 // This can be useful when you need to establish connections to databases or third-party services,
 // setup app components, compile assets, or any thing you need to do between starting Revel and accepting connections.
 //
-func OnAppStart(f func()) {
-	startupHooks = append(startupHooks, f)
+func OnAppStart(f func(), order ...int) {
+	o := 1
```

Comment:
That's true. We can assume all hooks will run at the same level just as they do currently and can be changed to different priority if needed. Should be good for backward compatibility.